libs/core/src/lib/slider/slider.component.ts

```diff
@@ -713,9 +713,13 @@ export class SliderComponent implements OnInit, OnChanges, OnDestroy, ControlVal
 
     /** @hidden Rtl change subscription */
     private _subscribeToRtl(): void {
-        this._isRtl = this._rtlService?.rtl.getValue();
+        if (!this._rtlService) {
+            return;
+        }
+
+        this._isRtl = this._rtlService.rtl.getValue();
```

Comment:
`this._rtlService.rtl` is a behavior subject so there is no need for this operation